src/services/group/group.service.ts

```diff
@@ -0,0 +1,77 @@
+import GroupService from './group.interface';
+import { Group, InternalError, ErrorType, NotFoundError, Person } from '../../models';
+import { GroupDao } from '../../data-access';
+import { PersonService } from '../person';
+
+export class GroupServiceImpl implements GroupService {
+  private readonly groupDao: GroupDao;
+  private readonly personService: PersonService;
+
+  constructor(groupDao: GroupDao, personService: PersonService) {
+    this.groupDao = groupDao;
+    this.personService = personService;
+  }
+
+  getById(id: string): Promise<Group> {
+    return this.groupDao
+      .getById(id)
+      .then((person) => {
+        if (!person) {
+          throw new NotFoundError(`Group with id ${id} was not found`);
+        }
+        return person;
+      })
+      .then(convertGroupFromEntity)
+      .catch(() => {
+        throw new NotFoundError(`Group with id ${id} was not found`);
+      });
+  }
+  create(group: Group): Promise<Group> {
+    return this.groupDao.create(group).then(() => group);
```

Comment:
You return an object the same as object in parameters. Please return which was created as a result of this.groupDao.create(group)

Reply:
fixed